README.md

```diff
@@ -57,7 +57,7 @@ Now that you have access to the `h1#header` element, use Javascript to change th
 
 **YOUR NOTES**
 ```
-
+Looking at a combination of the properties in console.dir(header) and google results, figure out how to change the color property of the style attribute of header to a new color. In this case, just .style.color.
```

Comment:
yay console.dir! if you feel like going down a rabbit hole, here's an [extensive list](https://developer.mozilla.org/en-US/docs/Web/API/console) of the `console` object's methods